inference-engine/cmake/vpu_dependencies.cmake

```diff
@@ -191,7 +191,7 @@ function(add_vpu_compile_custom_kernels)
                     "SHAVE_MA2X8XLIBS_DIR=${VPU_CLC_MA2X8X}/lib"
                     "SHAVE_MOVIASM_DIR=${VPU_CLC_MA2X8X}/bin"
                     "SHAVE_MYRIAD_LD_DIR=${VPU_CLC_MA2X8X}/bin"
-                ${VPU_CLC_MA2X8X_COMMAND} --strip-binary-header ${cl_file} -o ${out_file}
+                ${VPU_CLC_MA2X8X_COMMAND} --strip-binary-header -w ${cl_file} -o ${out_file}
```

Comment:
why do we need `-w`?

Reply:
Trying to disable clc warnings

Reply:
`-w` might also disable some useful of them...

Reply:
anyway this is not great for everybody to see OCL warnings if we are not planning to fix them

Reply:
it might be some useful for being fixed by the customer like unused parameter or possible aliasing. Hiding then implicitly would cause more pain rather than the strange one you've reported

Reply:
agreed